packages/devel/talloc/config/talloc-cache.txt

```diff
@@ -42,7 +42,7 @@ Checking for a 64-bit host to support lmdb: NO
 Checking value of GNUTLS_CIPHER_AES_128_CFB8: OK
 Checking value of GNUTLS_MAC_AES_CMAC_128: OK
 Checking whether fcntl supports flags to send direct I/O availability signals: OK
-Checking whether fcntl supports setting/geting hints: OK
+Checking whether fcntl supports setting/getting hints: OK
```

Comment:
This is code. Does this work? What is upstream?

Reply:
Doesn't exist upsteam. Reverted.